frontend/src/metabase/dashboard/containers/DashboardApp/DashboardApp.tsx

```diff
@@ -101,12 +106,24 @@ const mapDispatchToProps = {
 const connector = connect(mapStateToProps, mapDispatchToProps);
 type ReduxProps = ConnectedProps<typeof connector>;
 
-type DashboardAppProps = OwnProps & ReduxProps;
+type DashboardAppProps = OwnProps & ReduxProps & WithRouterProps;
 
 const DashboardApp = (props: DashboardAppProps) => {
   const { dashboard, isRunning, isLoadingComplete, isEditing, isDirty, route } =
     props;
 
+  const {
```

Comment:
this destructuring was some lines below, I just added `location` and moved it here so that I can access them right away